weekly planner/styles.css

```diff
@@ -0,0 +1,113 @@
+:root {
```

Comment:
good starting point for css, but we will need to change eventually to standardize across weekly and monthly view